package.json

```diff
@@ -13,7 +13,7 @@
     "test:coverage": "node node_modules/istanbul/lib/cli.js cover -e .ts  -x \"*.d.ts\" -x \"*.spec.ts\" node_modules/jasmine/bin/jasmine.js -- JASMINE_CONFIG_PATH=jasmine.json",
     "lint": "tslint --format stylish --force src/**/*.ts",
     "watch": "gulp watch",
-    "clean": "gulp clean"
+    "clean": "rm ./release -rf"
```

Comment:
I think the reason I used rimraf was to be able to run tests on Windows. Dunno state of things now…?